### PR TITLE
Create kab.yml

### PR DESCRIPTION
**Pull request by mozillakab**
Kabyle language file for rails-i18n

### CODE REVIEW

**Comment by BoFFire** [comment]
Need some check. It didn't pass the check.

File: rails/locale/kab.yml [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+fr:
+  activerecord:
+    errors:
+      messages:
+        record_invalid: 'Asentem ur yeddi aya : %{errors}'
```
Comment (BoFFire) [inline rails/locale/kab.yml]: "Asentem ur yeddi aya : %{errors}"
guillemets

File: rails/locale/kab.yml [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
+fr:
```
Comment (kuroda) [inline rails/locale/kab.yml]: This line must be `kab:`.

**Comment by digitalfrost** [comment]
Fails with

```

  1) I18n backend reloading when called twice doesn't produce warnings
     Failure/Error: errors.should == ''
     
       expected: ""
            got: nil (using ==)
     # ./spec/integration/backend_reload_spec.rb:21:in `block (3 levels) in <top (required)>'
```

**Pull request closed without merging**